src/magic_folder/test/__init__.py

```diff
@@ -62,6 +62,11 @@ def _configure_hypothesis():
     settings.register_profile(
         "magic-folder-fast",
         max_examples=1,
+        # see magic-folder-ci profile below for justification
+        suppress_health_check=[
+            HealthCheck.too_slow,
+        ],
+        deadline=None,
```

Comment:
I'm not sure this makes sense?

Reply:
I did this to match with the `-ci` one so that the only difference between the two is the number of examples run

Reply:
My thinking is that we want `magic-folder-fast` to be fast, and if something is triggering this health check, it isn't fast.

Reply:
Moved to PR: https://github.com/LeastAuthority/magic-folder/pull/361